frontend/src/metabase-types/api/database.ts

```diff
@@ -26,3 +26,9 @@ export interface Database {
   // Only appears in  GET /api/database/:id
   "can-manage"?: boolean;
 }
+
+export interface DatabaseData {
+  id?: DatabaseId;
+  engine: string | null;
+  details: Record<string, unknown>;
+}
```

Comment:
Not sure if `metabase-types` is a good place for form-specific types. Should we keep it under `metabase/databases/types` or something?

Reply:
Fixed, moved to local types